<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkyboxEditDialog</class>
 <widget class="QDialog" name="SkyboxEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sky box edit</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Negative Y:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QPushButton" name="pushButton_4">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="negXEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QLineEdit" name="posYEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Negative X:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QLineEdit" name="negZEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QLineEdit" name="posZEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Positive Z: </string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QLineEdit" name="negYEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="pushButton_6">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Positive Y: </string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Positive X: </string>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QPushButton" name="pushButton_5">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="posXEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Negative Z:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QPushButton" name="pushButton_7">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QLineEdit" name="nameEdit"/>
   </item>
   <item row="6" column="3">
    <widget class="QPushButton" name="pushButton_8">
     <property name="text">
      <string>Autopick</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>SkyboxEditDialog</receiver>
   <slot>onPosXClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>484</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>SkyboxEditDialog</receiver>
   <slot>onNegXClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>484</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>SkyboxEditDialog</receiver>
   <slot>onPosYClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>484</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>SkyboxEditDialog</receiver>
   <slot>onNegYClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>484</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>SkyboxEditDialog</receiver>
   <slot>onPosZClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>484</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>SkyboxEditDialog</receiver>
   <slot>onNegZClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>484</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>396</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>SkyboxEditDialog</receiver>
   <slot>onSaveClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>484</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>485</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_8</sender>
   <signal>clicked()</signal>
   <receiver>SkyboxEditDialog</receiver>
   <slot>onAutoPick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onPosXClicked()</slot>
  <slot>onNegXClicked()</slot>
  <slot>onPosYClicked()</slot>
  <slot>onNegYClicked()</slot>
  <slot>onPosZClicked()</slot>
  <slot>onNegZClicked()</slot>
  <slot>onSaveClicked()</slot>
  <slot>onPathClicked()</slot>
  <slot>onAutoPick()</slot>
 </slots>
</ui>
